Uzira.rte/Devices/Weapons/Crossbow/CrossbowBolt.lua

```diff
@@ -0,0 +1,53 @@
+function Create(self)
+	self.Sharpness = self.Sharpness * RangeRand(0.9, 1.1);
+	self.width = math.ceil(self:GetSpriteWidth() * 0.5);
+end
+
+function OnCollideWithMO(self, mo, rootMO)
+	if not self.target then
+		local trajectoryScalar = math.abs(math.cos(self.RotAngle - self.Vel.AbsRadAngle));
+
+		local dots = math.sqrt(self.PrevVel.Magnitude);
+		local trace = Vector((self.PrevVel.Magnitude * rte.PxTravelledPerFrame + self.width) * trajectoryScalar * self.FlipFactor, 0):RadRotate(self.PrevRotAngle);
+		for i = 1, dots do
+			local checkPos = self.PrevPos + trace * (i/dots);
+			local checkPix = SceneMan:GetMOIDPixel(checkPos.X, checkPos.Y);
+			if checkPix ~= rte.NoMOID then
+				local foundMO = MovableMan:GetMOFromID(checkPix);
+				if self.Mass * self.PrevVel.Magnitude * self.Sharpness > foundMO.Material.StructuralIntegrity then
+					checkPos = checkPos + SceneMan:ShortestDistance(checkPos, self.PrevPos, SceneMan.SceneWrapsX):SetMagnitude(self.width * 0.5);
+					self.target = ToMOSRotating(foundMO);
+					local dist = SceneMan:ShortestDistance(self.target.Pos, checkPos, SceneMan.SceneWrapsX);
+					local stickOffset = Vector(dist.X * self.target.FlipFactor, dist.Y):RadRotate(-self.target.RotAngle * self.target.FlipFactor);
+
+					local woundName = self.target:GetEntryWoundPresetName();
```

Comment:
Might be possible to clone the entry wound instead of using pressetname and create here. Investigate

Reply:
Would need source changes. Not gonna bother.